packages/jaeger-ui/src/components/TracePage/TracePageHeader/AltViewOptions.tsx

```diff
@@ -76,17 +76,20 @@ export default function AltViewOptions(props: Props) {
     onTraceViewChange(item);
   };
 
-  const menu = (
-    <Menu>
-      {MENU_ITEMS.filter(item => item.viewType !== viewType).map(item => (
-        <Menu.Item key={item.viewType}>
-          <a onClick={() => handleSelectView(item.viewType)} role="button">
-            {item.label}
-          </a>
-        </Menu.Item>
-      ))}
-      {!disableJsonView && (
-        <Menu.Item>
+  const dropdownItems = [
+    ...MENU_ITEMS.filter(item => item.viewType !== viewType).map(item => ({
+      label: (
+        <a onClick={() => handleSelectView(item.viewType)} role="button">
```

Comment:
Yes actually. Previously it needed a `Menu` wrapper for the dropdown overlay, but since v4.24.0 it no longer needs that.
That doesn't mean, there is a change in functionality. The items are still clickable all along that rectangle.

Here's a link to the official ant-design documentation which compares both of these usages: [Link](https://4x.ant.design/components/dropdown/#:~:text=const%20menu%20%3D%20(%0A%20%20%3CMenu%3E%0A%20%20%20%20%3CMenu.Item%3Eitem%201%3C/Menu.Item%3E%0A%20%20%20%20%3CMenu.Item%3Eitem%202%3C/Menu.Item%3E%0A%20%20%3C/Menu%3E%0A)%3B%0Areturn%20(%0A%20%20%3CDropdown%20overlay%3D%7Bmenu%7D%3E%0A%20%20%20%20%3Ca%3EHover%20me%3C/a%3E%0A%20%20%3C/Dropdown%3E%0A)%3B)